webservices/flow.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+import networkx as nx
+
+here, _ = os.path.split(__file__)
+home = os.path.join(here, os.pardir)
+script_path = os.path.join(home, 'data', 'sql_updates')
+
+def get_graph():
+    """Build a `DiGraph` that captures dependencies between database migration
+    tasks. Each node represents a migration script, and each edge represents
+    a dependency between tasks. Tasks can be ordered using topological sort.
+    """
```

Comment:
I am confused, if we add a new script, even with no dependencies, it won't be included in the `update_schemas` command if it is not in this function, or am I missing something?